src/renderer/components/+preferences/terminal.tsx

```diff
@@ -88,11 +95,11 @@ const NonInjectedTerminal = observer(({
         </section>
         <section>
           <SubTitle title="Font family" />
-          <Input
-            theme="round-black"
-            type="text"
+          <Select
+            themeName="lens"
             value={userStore.terminalConfig.fontFamily}
-            onChange={(value) => userStore.terminalConfig.fontFamily = value}
+            options={supportedCustomFonts}
```

Comment:
Minor: It would be cool if the labels for these options were in the font that they represent.

Reply:
Good idea. Gonna take a look soonish. 
_(since all fonts are preloaded and ready to use it's easy-picky addition)_

Reply:
<img width="909" alt="Screenshot 2022-05-25 at 16 48 44" src="https://user-images.githubusercontent.com/6377066/170278648-75ff1e2a-c984-45f5-86a0-aa0957a0c8ff.png">